apps/simplex-chat/Simplex/Chat/protocol.md

```diff
@@ -0,0 +1,59 @@
+# Chat protocol
+
+## Message syntax
+
+The syntax of the message inside agent MSG:
+
+```abnf
+agentMessageBody = message / msgContinuation
+message = [chatMsgId] SP msgEvent SP [parameters] SP msgBody SP
```

Comment:
why have `SP` in the end of `message`? wouldn't surrounding delimiters on an envelope would be enough for parsing?

Reply:
at the point of parsing the envelop is gone - it's just `MSG MsgMeta MsgBody`. So again, it's easier to parse when this delimiter is allowed (but it is currently optional anyway)